client/package.json

```diff
@@ -121,12 +133,13 @@
   },
   "sideEffects": false,
   "devDependencies": {
-    "@storybook/addon-a11y": "^6.1.21",
-    "@storybook/addon-actions": "^6.1.21",
+    "@storybook/addon-a11y": "^6.2",
+    "@storybook/addon-actions": "^6.2",
     "@storybook/addon-console": "^1.2.3",
-    "@storybook/addon-essentials": "^6.1.21",
-    "@storybook/addon-links": "^6.1.21",
-    "@storybook/react": "^6.1.21",
+    "@storybook/addon-essentials": "^6.2",
+    "@storybook/addon-links": "^6.2",
+    "@storybook/builder-webpack5": "^6.2",
+    "@storybook/react": "^6.2",
```

Comment:
storybook 6.1 uses webpack 4, which had [compatibility problems](https://github.com/storybookjs/storybook/issues/14157) with `ts-loader`. After adding `@storybook/builder-webpack5` and upgrading the others to 6.2 (and adding some polyfills above), I got storybook to work.